UI/RemoteISOScreen.cpp

```diff
@@ -598,8 +600,8 @@ RemoteISOBrowseScreen::RemoteISOBrowseScreen(const std::vector<std::string> &gam
 void RemoteISOBrowseScreen::CreateViews() {
 	bool vertical = UseVerticalLayout();
 
-	I18NCategory *mm = GetI18NCategory("MainMenu");
 	I18NCategory *di = GetI18NCategory("Dialog");
+	I18NCategory *n = GetI18NCategory("Networking");
```

Comment:
This definitely is not related to "networking", and if someone translated it thinking it was, they'd probably be translating it wrong.

Even if you disable game netplay, this functionality still works.  This functionality is more for when you want to play a game on your Android tablet, but you want to use the ISO that is on your PC.  It has nothing to do with netplay, and I don't want any of the UI to suggest otherwise - that would only confuse people.

-[Unknown]